services/bundle_analysis.py

```diff
@@ -123,6 +123,41 @@ async def initialize_and_save_report(
             db_session.flush()
         return commit_report
 
```

Comment:
This file is the only real change to this PR, everything else is ruff generated fixes.